.tmux.conf

```diff
@@ -177,9 +177,16 @@ run 'cut -c3- ~/.tmux.conf | sh -s apply_configuration'
 #       tmux_conf_theme_pane_border_fg=${tmux_conf_theme_pane_border_fg:-colour238}               # light gray
 #       tmux_conf_theme_pane_active_border_fg=${tmux_conf_theme_pane_active_border_fg:-colour39}  # light blue
 #
-#       tmux set -g pane-border-style fg=$tmux_conf_theme_pane_border_fg \; set -g pane-active-border-style fg=$tmux_conf_theme_pane_active_border_fg
-#       #uncomment for fat borders
-#       #tmux set -ga pane-border-style bg=$tmux_conf_theme_pane_border_fg \; set -ga pane-active-border-style bg=$tmux_conf_theme_pane_active_border_fg
+#       # if tmux_conf_theme_pane_border_bg is not defined
```

Comment:
Can you please avoid that `if`, `else`, `fi` dance by defining

```
tmux_conf_theme_pane_border_bg=default
tmux_conf_theme_pane_active_border_bg=default
```
And always call
```
tmux set -g pane-border-style fg=$tmux_conf_theme_pane_border_fg,bg=$tmux_conf_theme_pane_border_bg \; set -g pane-active-border-style fg=$tmux_conf_theme_pane_active_border_fg,bg=$tmux_conf_theme_pane_active_border_bg
```

Also, as far as configuration goes, I'm still unsure on how I want to expose theme customization. Instead of having `_fg` and `_bg` variables, maybe it's simple to have 2 `tmux_conf_theme_pane_border_` and `tmux_conf_theme_pane_active_border` along with a `tmux_conf_theme_pane_fat_border` boolean.